po/ca.po

```diff
@@ -0,0 +1,1483 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
```

Comment:
@webolot: Maybe you want to adjust these lines appropriately? :wink:

PS: Congratulations to your first successful PR and thank you for your contribution!

Reply:
Done! Sorry, I'm very new on this! ;-)